ImperatorToCk3Tests/ImperatorWorldTests/Provinces/ProvincesTests.cpp

```diff
@@ -0,0 +1,125 @@
+#include "gtest/gtest.h"
+#include <sstream>
+
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Provinces.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Province.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Pops.h"
+#include "../ImperatorToCK3/Source/Imperator/Provinces/Pop.h"
+
+TEST(ImperatorWorld_ProvincesTests, provincesDefaultToEmpty)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Provinces provinces(input);
+
+	ASSERT_TRUE(provinces.getProvinces().empty());
+}
+
+TEST(ImperatorWorld_ProvincesTests, provincesCanBeLoaded)
```

Comment:
Also add test for the none form here.

Reply:
There was no case of id=none in Provinces, did you mean Families?

Reply:
Added (for Families) ✔️